.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "third_party/abseil-cpp"]
+	path = third_party/abseil-cpp
+	url = https://github.com/abseil/abseil-cpp.git
```

Comment:
I'd prefer for this dependency to be isolated to the `iree-prof-tools` directory. This iree-samples repo has many different subprojects each either their own deps. Adding a top level git submodule like this would involve aligning on a common version across each subproject and also inflicts `git submodule` costs for each.

You can see the https://github.com/iree-org/iree-samples/tree/main/cpp folder for one way to achieve that:
* CMake FetchContent is used for the IREE repo: https://github.com/iree-org/iree-samples/blob/eb87773e01377873964d2389447e8930f8f8786f/cpp/CMakeLists.txt#L18-L46
* `find_package` is used for other deps: https://github.com/iree-org/iree-samples/blob/eb87773e01377873964d2389447e8930f8f8786f/cpp/vision_inference/CMakeLists.txt#L13-L19

FetchContent for the IREE repo may not be a good fit given the usage style, but I think absl could still be included differently.

Reply:
This is a great comment. We will sort out.